klog.go

```diff
@@ -768,7 +768,7 @@ func (l *loggingT) errorS(err error, loggr logr.Logger, msg string, keysAndValue
 // if loggr is specified, will call loggr.Info, otherwise output with logging module.
 func (l *loggingT) infoS(loggr logr.Logger, msg string, keysAndValues ...interface{}) {
 	if loggr != nil {
-		loggr.Info(msg, keysAndValues)
+		loggr.Info(msg, keysAndValues...)
```

Comment:
isn't this a problem in line 762 as well? can you please scan if there are other spots?

Reply:
you're right @dims - missed that one. Did a scan, but couldn't find any more. Pull req updated with new commit.